action.yml

```diff
@@ -2,10 +2,24 @@ name: 'Your name here'
 description: 'Provide a description here'
 author: 'Your name or organization here'
 inputs:
-  milliseconds: # change this
+  location:
```

Comment:
at some point we want to be able to support other package types (e.g. `cargo`). Maybe we should rename this to `npm_package_root` to indicate that it's only for an npm package?